common/app/model/meta.scala

```diff
@@ -59,12 +59,26 @@ object Fields {
       displayHint = apiContent.fields.flatMap(_.displayHint).getOrElse(""),
       isLive = apiContent.fields.flatMap(_.liveBloggingNow).getOrElse(false),
       sensitive = apiContent.fields.flatMap(_.sensitive),
-      shouldHideReaderRevenue = apiContent.fields.flatMap(_.shouldHideReaderRevenue),
+      shouldHideReaderRevenue = shouldHideReaderRevenue(apiContent),
       legallySensitive = apiContent.fields.flatMap(_.legallySensitive),
       firstPublicationDate = apiContent.fields.flatMap(_.firstPublicationDate).map(_.toJodaDateTime),
       lang = apiContent.fields.flatMap(_.lang)
     )
   }
+
+  private def shouldHideReaderRevenue(apiContent: contentapi.Content): Option[Boolean] = {
+    val cutoffDate = new DateTime("2017-07-03T12:00:00.000Z")
```

Comment:
I take @LATaylor-guardian's point that the cutoff date encodes the fact that the use of `isSensitive` is a historical artefact. But it is a bit ugly and also means I need to co-ordinate this PR with the [Composer PR](https://github.com/guardian/flexible-content/pull/2835/files). I'm just wondering, when you say:

> We can ensure on our side that it will default to false

Would this actually be covered by the [Composer PR](https://github.com/guardian/flexible-content/pull/2835/files)? Meaning, it looks to me like once that's deployed, all new Composer-created content will have this flag set to false by default, or true if the box is ticked.

Or is there content which ends up in CAPI but is not created in Composer, and you would need to default it for that too (and you don't currently)?